.github/workflows/test_generator_backend.yml

```
@@ -0,0 +1,98 @@
+
+jobs:
+  test-generator-backend:
+    name: Generate Backend Tests
+    runs-on: ubuntu-latest
+    # Définir ce job comme requis pour les autres workflows
+    outputs:
+      completed: ${{ steps.set-output.outputs.completed }}
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)